cypress/e2e/v1.0/journeys/drug-use/drug-use.noninjectable.morethan6months.cy.ts

```diff
@@ -0,0 +1,126 @@
+describe('Origin: /drug-use', () => {
```

Comment:
We tend to nest the `describe`s to better illustrate the full user journey

Reply:
I've given this a go in [321c26d](https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/828/commits/321c26d82730e98e5cc1888c7b0e63b92f7b5cb1)